WpfSampleDi/MainWindow.xaml.cs

```diff
@@ -1,24 +1,19 @@
-﻿using System.Windows;
-using WpfSampleDi.Service;
+﻿using WpfSampleDi.ViewModels;
 
-namespace WpfSampleDi
+namespace WpfSampleDi;
+
+/// <summary>
+/// Interaction logic for MainWindow.xaml
+/// </summary>
+public partial class MainWindow
 {
-    /// <summary>
-    /// Interaction logic for MainWindow.xaml
-    /// </summary>
-    public partial class MainWindow : Window
+    public MainWindow()
     {
-        public readonly ILogBase LogBase;

```

Comment:
You don't really want to expose your logger like that. Just inject it into constructor of every class that needs it, `ServiceProvider` will take care of instantiating the correct logger for you.